vmm/src/device_manager.rs

```diff
@@ -3405,6 +3412,27 @@ impl DeviceManager {
 
         counters
     }
+
+    pub fn resize_balloon(&mut self, size: u64) -> DeviceManagerResult<()> {
+        if let Some(balloon) = &self.balloon {
+            return balloon
+                .lock()
+                .unwrap()
+                .resize(size)
+                .map_err(DeviceManagerError::VirtioBalloonResize);
+        }
+
+        warn!("No balloon setup: Can't resize the balloon");
+        Err(DeviceManagerError::MissingVirtioBalloon)
+    }
+
+    pub fn balloon_size(&self) -> u64 {
```

Comment:
The original name is get_balloon_actual.
Change it to balloon_size is unclear.

Reply:
That's just simpler. We don't need the `get_` prefix since we understand we want to get the balloon size out of this function.
And using actual is not needed since the balloon size asked by the user is not something that we store or carry anywhere. That means that by default, when we ask for the balloon size, the function will give us the actual size in use by the guest. I think it's important to keep names simple when they don't introduce any confusion.

Reply:
Actually, I cannot understand this part.  :(

In the out side, it is memory_actual_size.  In the balloon device, it is "get_actual".  And in the "pub struct BalloonConfig {", there is another "size".

It is a internal function name, if you think it is clear, I am OK with this.